src/equations/lattice_boltzmann_2d.jl

```diff
@@ -119,9 +123,10 @@ function LatticeBoltzmannEquations2D(; Ma, Re, collision_op = collision_bgk,
     Ma, Re, c, L, rho0, u0, nu = promote(Ma, Re, c, L, rho0, u0, nu)
 
     # Source for weights and speeds: [4] in the docstring above
-    weights = SVector(1 / 9, 1 / 9, 1 / 9, 1 / 9, 1 / 36, 1 / 36, 1 / 36, 1 / 36, 4 / 9)
-    v_alpha1 = SVector(c, 0, -c, 0, c, -c, -c, c, 0)
-    v_alpha2 = SVector(0, c, 0, -c, c, c, -c, -c, 0)
+    weights = SVector{9, RealT}(1 / 9, 1 / 9, 1 / 9, 1 / 9, 1 / 36, 1 / 36, 1 / 36,
+                                1 / 36, 4 / 9)
+    v_alpha1 = SVector{9, RealT}(c, 0, -c, 0, c, -c, -c, c, 0)
+    v_alpha2 = SVector{9, RealT}(0, c, 0, -c, c, c, -c, -c, 0)
```

Comment:
@ranocha Do you think this is a good design?

Reply:
Yes, this looks reasonable to me 👍